class1/exercises.livemd

```diff
@@ -95,8 +116,24 @@ Tip: You can calculate score for each group in separate function and pipe them t
 
 ```elixir
 defmodule Scrabble do
+  defp get_letter_value(letter) do
+    cond do
```

Comment:
You have correctly used `cond do` here.
Another approach would be to create function with guards, 
```elixir
defp get_letter_value(letter) when letter in ["A", "E",...], do: 1
# or
defp get_letter_value(letter) when "AEI" =~ letter, do: 1
```
In Elixir you can have many function with the same name and even with the same name and different amount of arguments.

When functions is called Elixir will try to match given function call to what is implemented taking into accounts guards or pattern match. And if no match is found then `FunctionClauseError` is raised

Reply:
It looks like multiple functions with the same name is the proper way of Elixir program implementation. I'll try to implement methods in that manner